Docs: Getting support documentation reworked (#5761)
WeblateOrg/weblate: docs/admin/support.rst

===========================

*Weblate* is copylefted libre software with community support.
Subscribers receive priority support at no extra charge.
Prepaid help packages are available for everyone.

> Find info about current support offerings at https://weblate.org/support.

Author: There was some escaping for URLs ending in dots in another PR. Maybe needed here?

Weblate is copylefted libre software with community support.
Subscribers receive priority support at no extra charge. Prepaid help packages are
available for everyone. You can find more info about current support

> offerings at `https://weblate.org/support <https://weblate.org/support/>`_.

Reviewer: What is point in duplicating links? See my comments on https://github.com/WeblateOrg/weblate/pull/5742

Author: They currently look like 
![bilde](https://user-images.githubusercontent.com/13802408/112740211-a0157c00-8f6a-11eb-97bd-808b43c5c563.png)
Not sure if the URL only makes it a hotlink, but marking it up makes it easy to change the link later on if needed.
The last / makes it easy to see what part is the actual link, and which part is the displayed name.

I'll change it to URLs only.

Reviewer 2: corrected, changed to URLs only

> offerings at `https://weblate.org/support <https://weblate.org/support/>`_.

Author: ```suggestion
offerings at https://weblate.org/support.
```

.. _support-data:

> Info sent to and shown on Weblate.org

Author: ```suggestion
Info sent to and (partially) shown on Weblate.org
```

> -------------------------------------

Reviewer: Not all data is shown on weblate.org.

Reviewer 2: solved by rewrite.

> -------------------------------------

Author: ```suggestion
-----------------------------------------------
```

> -----------------------------------------------

Author: ```suggestion
-------------------------------------------------
```

.. _discover-weblate:

> Discover
> --------

Reviewer: ```suggestion
Discover Weblate
----------------
```

.. versionadded:: 4.5.2

.. note::

   This feature is currently in early beta.

*Discover* is an opt-in service making it easier for users to find
other Weblate sites and communities. Users can browse registered sites on

> https://weblate.org/discover and find

Reviewer: ```suggestion
https://weblate.org/discover/ and find
```

projects to contribute to there.

Getting listed
++++++++++++++

.. note::

   Make sure you are OK with publishing some of the info about your
   Weblate described in :ref:`support-data` for use on *Discover*.

List your server directly from the management panel if you have an active
support subscription (:ref:`activate-support`):

.. image:: /screenshots/support-discovery.png

All steps for listing a server without a support contract:

1. Register yourself at https://weblate.org/user/
2. Register your Weblate site at https://weblate.org/subscription/discovery

> 3. Confirm you you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :

Reviewer: ```suggestion
3. Confirm you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
```

.. image:: /screenshots/support-discovery.png

.. _customize-discover:

Listing description
+++++++++++++++++++

Add a text and an image (570 x 260 pixels) about your Weblate site

> from https://weblate.org/user.

Author: ```suggestion
from <https://weblate.org/user/>.
```

.. versionadded:: 4.5.2

*Discover* is an opt-in service making it easier for users to find
other Weblate sites and communities.
Users can browse registered sites and find projects to contribute to on

> https://weblate.org/discover/.

Author: There was some escaping for URLs ending in dots in another PR. Maybe needed here?

Reviewer: https://weblate--5761.org.readthedocs.build/en/5761/ shows how this gets rendered. This is available for all pull request, look for Read the docs in checks

   This feature is currently in early beta.

Discover Weblate is an opt-in service making it easier for users to find
other Weblate sites and communities. Users can browse registered sites on

> `https://weblate.org/discover <https://weblate.org/discover/>`_, and find

Author: ```suggestion
https://weblate.org/discover/ and find
```

projects to contribute to there.

Getting listed
++++++++++++++

.. note::

   Make sure you are OK with publishing the info about your Weblate

>    described in :ref:`support-data` for use on Discover Weblate.

Reviewer: Not all submitted data is published.

List your server directly from the management panel if you have an active
support subscription (:ref:`activate-support`):

.. image:: /images/support-discovery.png

> Listing your server manually:

Reviewer: This is not a more manually that with a subscription, it just needs additional steps.

> 1. Register yourself at `https://weblate.org/user <https://weblate.org/user/>`_

Author: ```suggestion
1. Register yourself at https://weblate.org/user/
```

> 2. Register your Weblate site at https://weblate.org/subscription/discovery

Reviewer: ```suggestion
2. Register your Weblate site at https://weblate.org/subscription/discovery/
```

> 2. Register your Weblate site at `https://weblate.org/subscription/discovery <https://weblate.org/subscription/discovery/>`_

Author: ```suggestion
2. Register your Weblate site at https://weblate.org/subscription/discovery
```

> 3. Confirm you want to activate the service from your Weblate, and turn on "Discovery listing" from its management page by clicking :guilabel:`Enable discovery`:

Author: ```suggestion
3. Confirm you you want your Weblate listed by clicking :guilabel:`Enable discovery` from its management page :
```

.. image:: /images/support-discovery.png

.. _customize-discover:

> Listing description

Author: This isn't great

+++++++++++++++++++

Add a text and an image (570 x 260 pixels) about your Weblate site

> from <https://weblate.org/user/>.

Author: This allows the ending dot?

Reviewer: Yes.

> from `https://weblate.org/user <https://weblate.org/user/>`_.

Author: ```suggestion
from https://weblate.org/user.
```
